### PR TITLE
Improve markdown explanation

### PR DESCRIPTION
**Pull request by bmesuere**
This pull request adds a basic markdown formatting description to our own documentation. We already linked to a cheatsheet on that page, but this was an extra step and in English.

### CODE REVIEW

File: en/references/exercise-description/README.md [hunk 1]
```diff
@@ -11,7 +11,69 @@ These features are available for both exercises and reading activities. They are
 
 ## Markdown
 
- For Dodona exercises and reading activities, we recommend using Markdown over HTML. Although Markdown is widely used on the internet, the format is not standardized. This led to existence of several Markdown "flavors". Dodona uses [kramdown Syntax](http://kramdown.gettalong.org/syntax.html) and the Kramdown parser to generate HTML from the Markdown source. You can find more information about basic Markdown formatting on this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+For Dodona exercises and reading activities, we recommend using Markdown over HTML. Descriptions for courses and series can also be written in Markdown. Markdown is a markup language like HTML, but a lot simpler. Below, you can find more information about the basic formatting and a list of all syntax in this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). The rest of this page contains specific extentions (in both Markdown and HTML) that are available in exercise descriptions on Dodona.
```
Comment (chvp) [inline en/references/exercise-description/README.md:14]:
```suggestion
For Dodona exercises and reading activities, we recommend using Markdown over HTML. Descriptions for courses and series can also be written in Markdown. Markdown is a markup language like HTML, but a lot simpler. Below, you can find more information about the basic formatting and a list of all syntax in this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). The rest of this page contains specific extensions (in both Markdown and HTML) that are available in exercise descriptions on Dodona.
```
(https://www.askdifference.com/extention-vs-extension/)

**Review by chvp — APPROVED** [review]

File: en/references/exercise-description/README.md [hunk 2]
```diff
@@ -11,7 +11,69 @@ These features are available for both exercises and reading activities. They are
...
+For Dodona exercises and reading activities, we recommend using Markdown over HTML. Descriptions for courses and series can also be written in Markdown. Markdown is a markup language like HTML, but a lot simpler. Below, you can find more information about the basic formatting and a list of all syntax in this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). The rest of this page contains specific extensions (in both Markdown and HTML) that are available in exercise descriptions on Dodona.
+
+### Headers
+
+In markdown, it is easy to use headers of different levels. For this, you simply put one or more hash symbols (`#`) in front of your title. Depending on the number of symbols used, a larger or smaller title will be shown.
```
Comment (niknetniko) [inline en/references/exercise-description/README.md:18]:
```suggestion
In Markdown, it is easy to use headers of different levels. For this, you simply put one or more hash symbols (`#`) in front of your title. Depending on the number of symbols used, a larger or smaller title will be shown.
```

I think we should consistently capitalise Markdown (or not), but most other occurrences seem to be capitalized (

File: en/references/exercise-description/README.md [hunk 3]
```diff
@@ -11,7 +11,69 @@ These features are available for both exercises and reading activities. They are
...
+## Level 2
+### Level 3
+#### Level 4
+##### Level 5
+###### Level 6
+```
+
+### Text formatting
+
+For simple text formatting, place tekst between stars (`*`). If one or more words are between single starts, the text will render in italics, if you use double stars, it wil render in bold. If you place words between double tilde signs (`~`), the text will be striked through. To format text as code, place the text between backticks (`` ` ``).
```
Comment (niknetniko) [inline en/references/exercise-description/README.md:31]:
```suggestion
For simple text formatting, place tekst between stars (`*`). If one or more words are between single starts, the text will render in italics, if you use double stars, it wil render in bold. If you place words between double tilde signs (`~`), the text will be struck through. To format text as code, place the text between backticks (`` ` ``).
```

I think the verb is `struck through`, while a text that has that style applied is called a `strikethrough` (according to wikidictionary at least https://en.wiktionary.org/wiki/strikethrough#English)

File: en/references/exercise-description/README.md [hunk 4]
```diff
@@ -11,7 +11,69 @@ These features are available for both exercises and reading activities. They are
...
+
+```markdown
+This is an example with words in *italics*, **bold**, and ~~striked through~~ and with a variable `a` in function `returnSolution()`.
+```
+
+This example results in this formatting: This is an example with words in *italics*, **bold**, and ~~striked through~~ and with a variable `a` in function `returnSolution()`.
```
Comment (niknetniko) [inline en/references/exercise-description/README.md:37]:
```suggestion
This example results in this formatting: This is an example with words in *italics*, **bold**, and ~~strikethrough~~ and with a variable `a` in function `returnSolution()`.
```

File: en/references/exercise-description/README.md [hunk 5]
```diff
@@ -11,7 +11,69 @@ These features are available for both exercises and reading activities. They are
...
+
+### Hyperlinks and images
+
+Hyperlinks and images are a little more complex. They consist out of two parts of which you place the first part between square brackets (`[]`) and the second part between round pbrackets (`()`). The first part is used for the words that need to be linked or the image description, the second part is used for the url of the page or image. To indicate you want to render an image, place an exclamation mark (`!`) in front of the first square bracket.
```
Comment (niknetniko) [inline en/references/exercise-description/README.md:41]:
```suggestion
Hyperlinks and images are a little more complex. They consist out of two parts of which you place the first part between square brackets (`[]`) and the second part between round brackets (`()`). The first part is used for the words that need to be linked or the image description, the second part is used for the URL of the page or image. To indicate you want to render an image, place an exclamation mark (`!`) in front of the first square bracket.
```

(Not sure about the URL/url, but I think it is often capitalised in English)

File: nl/references/exercise-description/README.md [hunk 6]
```diff
@@ -11,7 +11,68 @@ De features die op deze pagina beschreven worden zijn beschikbaar voor zowel oef
...
+Voor oefeningen en leesactiviteiten op Dodona raden we aan om de inhoud te schrijven in Markdown. Ook voor beschrijvingen van cursussen en reeksen kan Markdown gebruikt worden. Markdown is een soort opmaaktaal zoals HTML, maar dan een stuk eenvoudiger. Hieronder kan je de basisopmaak vinden, een lijst van alle syntax kan je vinden in deze [Markdown Cheatsheet pagina](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Verder op deze pagina staan specifieke uitbreidingen die werken binnen Dodona oefeningenbeschrijvingen.
+
+### Titels
+
+In markdown kan je eenvoudig titels van verschillende niveaus gebruiken. Je plaats hiervoor één of meerdere hekjes (`#`) voor je titel. Afhankelijk van het aantal hekjes zal je een grotere of kleinere titel krijgen.
```
Comment (niknetniko) [inline nl/references/exercise-description/README.md:18]:
```suggestion
In Markdown kan je eenvoudig titels van verschillende niveaus gebruiken. Je plaats hiervoor één of meerdere hekjes (`#`) voor je titel. Afhankelijk van het aantal hekjes zal je een grotere of kleinere titel krijgen.
```

File: nl/references/exercise-description/README.md [hunk 7]
```diff
@@ -11,7 +11,68 @@ De features die op deze pagina beschreven worden zijn beschikbaar voor zowel oef
...
+```
+
+Dit is een zin waarbij [deze woorden](https://google.com) linken naar Google.
+Hieronder staat het Dodona logo afgebeeld.
+
+![Dodona logo](https://dodona.ugent.be/icon.png)
+
+### Opsommingen
+
+Opsommingen werken heel natuurlijk in markdown. Afhankelijk of je een genummerde of een gewone opsomming wil, laat je je regel voorafgaan door `1. ` of `* `. Dit ziet er dan als volgt uit:
```
Comment (niknetniko) [inline nl/references/exercise-description/README.md:57]:
```suggestion
Opsommingen werken heel natuurlijk in Markdown. Afhankelijk of je een genummerde of een gewone opsomming wil, laat je je regel voorafgaan door `1. ` of `* `. Dit ziet er dan als volgt uit:
```

**Review by niknetniko — CHANGES_REQUESTED** [review]

**Comment by bmesuere** [comment]
Thanks, @niknetniko

**Pull request merged**